apps/system/utils/utils_cpuload.c

```diff
@@ -214,33 +205,25 @@ static void cpuload_print_normal(void)
 	/* Print titles */
 	printf("\n--------------------------------------------------\n");
 #ifdef CONFIG_SCHED_MULTI_CPULOAD
-	printf("Non-Zero CPU utilization (%ds), (%ds), (%ds)\n", CONFIG_SCHED_CPULOAD_TIMECONSTANT_SHORT, CONFIG_SCHED_CPULOAD_TIMECONSTANT_MID, CONFIG_SCHED_CPULOAD_TIMECONSTANT_LONG);
+	printf("Non-Zero CPU utilization trend (updated every %ds)\n", cpuload_interval);
 #else
-	printf("Non-Zero CPU utilization (%ds)\n", CONFIG_SCHED_CPULOAD_TIMECONSTANT);
+	printf("Non-Zero CPU utilization trend (updated every %ds)\n", cpuload_interval);
```

Comment:
The `ifdef` and the `else` cases are perfectly the same. Let's keep one line for this.